modules/openfast-library/src/FAST_Subs.f90

```diff
@@ -5723,12 +5708,12 @@ SUBROUTINE WrVTK_BasicMeshes(p_FAST, y_FAST, MeshMapData, ED, BD, AD, IfW, OpFM,
 
 
 ! Blades
-   IF ( p_FAST%CompAero == Module_AD ) THEN  ! These meshes may have airfoil data associated with nodes...
+   IF ( p_FAST%CompAero == Module_AD .and. ALLOCATED(AD%Input) ) THEN  ! These meshes may have airfoil data associated with nodes...
```

Comment:
Could this potentially fail with AD14 or AeroDisk in use?  If I remember correctly, `AD%Input` will be allocated even if AD15 isn't used, but things inside it wouldn't be.

Reply:
Never mind.  This is not an issue (reread logic above).

Reply:
The biggest concern is that if the code is aborting due to error and is trying to output the `DebugError` versions of the VTK files, we need to make sure things are allocated before using them. We should probably also check that `AD%Input(1)%rotors` and `AD%y%rotors` are allocated. I'll add that to another PR.